GoogleForCreators/web-stories-wp
# Drop targeting: preserve origin element link if available.

## Summary
Adjusts `combineElements` logic so that the origin elements link would be preserved.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
I
1. Add a shape and an image.
2. Add links to both (different links)
3. Drag the image into the shape
4. Verify that the link set to the image has been preserved.

II
1. Add a shape and a video
2. Add link to the shape
3. Now drag the video into the shape
4. Verify that the link of the shape has been preserved.

III
1. Add a video as a background
2. Add an image and assign link to it
3. Drag the image into the background
4. Unselect the background and then hover over it -- verify it doesn't display any link information.


---



Fixes #641
